WeatherToday/Sources/Data/NetworkAPI/RequestConfigurations/WeatherApiRequestConfigurations.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  WeatherApiRequestConfigurations.swift
+//  WeatherToday
+//
+//  Created by Taher on 10/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import Alamofire
```

Comment:
Please rearrange import by alphabetical order.

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/0df0b8e6ef1945dbb46c9bd130e52d369b861265)